modules/cluster-logging-curator-delete-index.adoc

```diff
@@ -47,6 +47,7 @@ The available parameters are:
 |`project_name`
 |The actual name of a project, such as *myapp-devel*. For {product-title} *operations*
 logs, use the name `.operations` as the project name.
+// TBD - Update from `.operations` to `infra` for https://bugzilla.redhat.com/show_bug.cgi?id=1873372?
```

Comment:
I am not sure if we can really drop or change in the curator configuration here. It looks dedicated to curate the old pre-4.5 indices `.project...` and `.operations...`. I would pass this to @jcantrill to answer.

Reply:
Good catch. I've removed that comment and confirmed this example is for pre-4.5 indices.